products/BellHybrid/apps/application-bell-meditation-timer/presenter/MeditationProgressPresenter.hpp

```diff
@@ -100,5 +85,24 @@ namespace app::meditation
         void abandon() override;
         void finish() override;
         void onBeforeShow() override;
+        void playGongSound() override;
+        void playEndSound() override;
+
+      private:
+        app::ApplicationCommon *app{nullptr};
+        settings::Settings *settings{nullptr};
+        std::unique_ptr<app::TimerWithCallbacks> timer;
+        std::unique_ptr<AbstractTimeModel> timeModel;
+        models::ChimeInterval &chimeIntervalModel;
+        models::Statistics &statisticsModel;
+        std::chrono::minutes duration;
+        std::chrono::seconds interval;
+
+        static constexpr auto endWindowTimeout = std::chrono::seconds{5};
```

Comment:
I think this can be moved to a .cpp file

Reply:
You're right, done!